.gitignore

```diff
@@ -1,5 +1,6 @@
 # dependencies
 node_modules/
+yarn.lock
```

Comment:
This probably also needs to be added to `.spraygun.js` as part of the replacements array as Matt mentioned on this [PR](https://github.com/carbonfive/spraygun-react/pull/105)